service.go

```diff
@@ -20,25 +20,26 @@ const (
 	WatchTimeOut = 90 * time.Second
 )
 
-// Config represents the configuration used to create a service.
-type Config struct {
+// ServiceConfig represents the configuration used to create a certificate TPR
+// service.
+type ServiceConfig struct {
 	// Dependencies.
 	K8sClient kubernetes.Interface
 	Logger    micrologger.Logger
 }
 
-// DefaultConfig provides a default configuration to create a new service by
-// best effort.
-func DefaultConfig() Config {
-	return Config{
+// DefaultServiceConfig provides a default configuration to create a new
+// certificate TPR service by best effort.
+func DefaultServiceConfig() ServiceConfig {
+	return ServiceConfig{
 		// Dependencies.
 		K8sClient: nil,
 		Logger:    nil,
 	}
 }
 
-// New creates a new configured service.
-func New(config Config) (*Service, error) {
+// NewService creates a new configured certificate TPR service.
+func NewService(config ServiceConfig) (*Service, error) {
```

Comment:
This is the `certificatetpr` repository. Using `New` means creating a new certificate TPR. What we are creating here is a certificate TPR service that is e.g. able to search for certificates. So `NewService` is more appropriate here.